js/main.js

```diff
@@ -163,22 +176,23 @@ function seeQuestions(results) {
   $p4.prepend($d);
 
   var $submitDiv = document.createElement('div');
-  $submitDiv.setAttribute('class', 'row center margin-top-submit');
-
+  $submitDiv.setAttribute('class', 'row center margin-top-submit flex-wrap');
   $questionContainer.appendChild($submitDiv);
 
   var $submit = document.createElement('button');
-
   $submit.textContent = 'Submit';
-
   $submit.setAttribute('class', 'submit');
 
   $submitDiv.appendChild($submit);
 
   $choiceDiv.addEventListener('click', clickAnswer);
-
   $submit.addEventListener('click', submitAnswer);
 
+  var $totalScore = document.createElement('div');
+  $totalScore.textContent = String(dataModel.player) + ' has answered ' + String(dataModel.playerObj[String(dataModel.player)]) + ' questions correctly.';
```

Comment:
You don't need to call the `String` function in this, it will be coerced into a string automatically because you are concatenating a string.